### DISCUSSION TITLE
[AIP-8] RoundImplementation: Decoupling features — Discussion + Comments

### DISCUSSION THREAD

**Discussion by nfrgosselin** [allo-protocol/aips#22] [AIPs in Review]
The initial draft of AIP-8, a proposal for decoupling features in the RoundImplementation contract, has been added to the AIP repo here: https://github.com/allo-protocol/aips/blob/aip-8-roundImplementation-decouple-features/AIPS/aip-8.md

Please post any feedback or discussion items related to this AIP in this discussion thread.

**Comment by lmcorbalan** [comment]
Wouldn't be a good idea to include in this AIP the changes related to the `AllocationStrategy` discussed on #11 ?

**Reply by nfrgosselin** [reply]
cc-ing in @KurtMerbeth and @thelostone-mc

@lmcorbalan are you referring to the re-naming or something else?
